setup.py

```diff
@@ -306,20 +336,20 @@ def cython_extensions_and_necessity():
 SETUP_REQUIRES = INSTALL_REQUIRES + (
```

Comment:
Should we remove `futures` and `enum34` from our install requirement @ line 332-333?

Reply:
Same story. I must have dropped a commit at some point.